receiver/JSONReceiver.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import json
+
+from aiokafka import AIOKafkaConsumer
+
+from receiver.MessageHandlingStrategy import MessageHandlingStrategy
+from receiver.consts import RATING_TOPIC_NAME, KAFKA_BROKER_URL
+from receiver.log import logger
+
+
+class JSONReceiver:
+    def __init__(self, handler_strategy: MessageHandlingStrategy) -> None:
+        self.handler_strategy = handler_strategy
+
+    def start(self) -> None:
+        asyncio.run(self.__consume_json_messages())
+
+    async def __consume_json_messages(self) -> None:
```

Comment:
```Suggestion
async def _consume_json_messages(self) -> None:
```
Private functions in python start with a single underscore.

Reply:
I remember protected being a single underscore and private being 2 underscores.